streaming/index.js

```diff
@@ -1177,7 +1177,7 @@ const startServer = async () => {
     const location = url.parse(req.url, true);
 
     req.requestId = uuid.v4();
-    req.remoteAddress = ws._socket.remoteAddress;
+    req.remoteAddress =  req.headers['x-forwarded-for'].split(',')[0].trim();
```

Comment:
There needs to be a fallback here in case the header is not present (ie, not using a reverse proxy)

Currently it will crash because `req.headers['x-forwarded-for']` will be empty.

Reply:
Added the fallback.